capstone/src/app/components/WishlistForm.tsx

```diff
@@ -62,31 +62,37 @@ export default function WishlistForm() {
   const removeItem = (index: number) => {
     const updatedWishlist = wishlist.filter((_, i) => i !== index);
     setWishlist(updatedWishlist);
-    localStorage.setItem('wishlist', JSON.stringify(updatedWishlist)); // Update localStorage
+    localStorage.setItem("wishlist", JSON.stringify(updatedWishlist)); // Update localStorage
```

Comment:
majority of these changes are just replacing the single quotes with double! I ran the code through a react format website to indent the code, beautify and such !